src/updates.py

```diff
@@ -99,15 +103,36 @@ def download(session, sub_names, res_urls, path):
     return sub_updates
 
 
-def main(session, sub_names, sorted_urls, path):
-    notice_urls, news_urls, res_urls = sorted_urls
-    print ("\t\t" + bold("**Nalanda**"))
-    unread_update = find_new(
-        session, sub_names, notice_urls, "Notices")
-    term_display(unread_update, "Notices", sub_names)
-    subject_news_url = get_news(session, sub_names, news_urls)
-    unread_update = find_new(
-        session, sub_names, subject_news_url, "News")
-    term_display(unread_update, "News", sub_names)
-    update_list = download(session, sub_names, res_urls, path)
-    term_display(update_list, "Lectures", sub_names, path)
+class main:
+    def __init__(self, session, sub_names, sorted_urls, path):
+        self.urls = sorted_urls
+        self.sion = session
+        self.s_name = sub_names
+        self.path = path
+        print("\t\t" + bold("**Nalanda**"))
+
+    def show_notice(self, reset=1):
```

Comment:
Don't write different functions when they have similar functionality. Add another parameter in the function and use it in function calls to differentiate.
Avoid DRY Code